### PR TITLE
Allow content view versions to be searched

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -1205,7 +1205,7 @@ def __init__(self, server_config=None, **kwargs):
         super(ContentUpload, self).__init__(server_config, **kwargs)
 
 
-class ContentViewVersion(Entity, EntityReadMixin, EntityDeleteMixin):
+class ContentViewVersion(Entity, EntityReadMixin, EntityDeleteMixin, EntitySearchMixin):
```
Comment (rochacbruno) [inline nailgun/entities.py]: Line is too long
`/nailgun/entities.py:1208:80: E501 line too long (88 > 79 characters)`

we keep a limit to 79 chars.

```python
class ContentViewVersion(
        Entity, 
        EntityReadMixin, 
        EntityDeleteMixin, 
        EntitySearchMixin):
    """A representation of a Content View Version non-entity."""
    def __init__(self, server_config=None, **kwargs):
```

**Comment by ehelms** [comment]
@rochacbruno updated and passing!

**Comment by rochacbruno** [comment]
ACK

**Comment by ehelms** [comment]
Once this is merged, if I could request a new release of Nailgun soonish so that I can update my Ansible module that would be awesome.

**Comment by omaciel** [comment]
@ehelms can you provide a working example of the new code 'in action'?

**Comment by ehelms** [comment]
@omaciel sure, I am using it to do a find of a particular content view version for a given environment for publish and promote:

https://github.com/ansible/ansible-modules-extras/pull/2450/files#diff-8d39f2f49b639d6c727a51dcd785425eR398

https://github.com/ansible/ansible-modules-extras/pull/2450/files#diff-8d39f2f49b639d6c727a51dcd785425eR410

**Comment by svtkachenko** [comment]
I mean read and delete mixins. Though it was already in that order :D

**Comment by ehelms** [comment]
@svtkachenko updated :)

**Pull request merged**